pandas-dev/pandas
# DOC: reverse rolling window (#38627)

This adds the suggested changes from closed pull request (#38627):
https://github.com/pandas-dev/pandas/pull/39091/files/80819f65402e46caa29ff7192b34d43f91c1a0ee
After updated commit.
- remove noqa
- change line back to single quotes
- rename `df1` to `reversed_df`
- add reference to `FixedForwardWindowIndexer`
- remove repeated `df` initialization in code example
- rename `df2` to `reversed_df`
- move code example

- [x] closes #38627
- [x] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing.html#code-standards) for how to run them
